AppImage-package/bundle.sh

```diff
@@ -1,86 +1,115 @@
 #!/bin/bash
+
 ########################################################################
 # Package the binaries built on Travis-CI as an AppImage
 # By Simon Peter 2016
 # For more information, see http://appimage.org/
 ########################################################################
+
 sudo apt-get -y install coreutils binutils python-tk
```

Comment:
This should be moved to the `.travis.yml` files with other `apt-get` calls, since this will be done without sudo in the future.